discovery-infra/test_infra/config/_cluster_config.py

```diff
@@ -0,0 +1,55 @@
+from dataclasses import dataclass
+
+from test_infra.config import EnvConfig
+from test_infra.helper_classes.cluster_config import ClusterConfig, ClusterInstallConfig
+
+
+@dataclass
+class DefaultInstallConfig(ClusterInstallConfig):
+
+    def __post_init__(self):
+        self.iso_download_path = self.iso_download_path \
+            if self.iso_download_path is not None else EnvConfig.get("iso_download_path")
+        self.iso_image_type = self.iso_image_type \
+            if self.iso_image_type is not None else EnvConfig.get("iso_image_type")
+        self.nodes_count = self.nodes_count \
+            if self.nodes_count is not None else EnvConfig.get("num_nodes")
+        self.vip_dhcp_allocation = self.vip_dhcp_allocation \
+            if self.vip_dhcp_allocation is not None else EnvConfig.get("vip_dhcp_allocation")
+        self.download_image = self.download_image \
+            if self.download_image is not None else EnvConfig.get("download_image")
```

Comment:
wdty, about removing download_image from here and leaving it as a parameter to `prepare_for_installation`?
This is not a test wide parameter, it is only relevant to `prepare_for_installation`